packages/ember-metal/lib/tags.js

```diff
@@ -19,11 +24,23 @@ export function tagFor(object, _meta) {
   }
 }
 
+function K() {}
+function ensureRunloop() {
+  if (!run) {
+    run = require('ember-metal/run_loop').default;
```

Comment:
It cannot. Within ember-metal, run_loop imports property_events, property_events imports tags, and tags needs to import run_loop. However, this creates a circular dependency. It had worked at one point because of a beneficial, coincidental ordering of evaluation. Using the imperative require lets us explicitly choose where the circle unwinds, instead of leaving it up to fate.